src/hoc/AppMargin.tsx

```diff
@@ -29,4 +32,16 @@ export const AppMargin = ({
   if (size === SizeofAppMargin.SMALL) {
     return <Box sx={{ ...sx, padding: { xs: `${py ? py : '0px'} 20px`, sm: `${py ? py : '0px'} 25px` } }}>{children}</Box>
   }
+  if (size === SizeofAppMargin.CUSTOM) {
+    return (
+      <Box
+        sx={{
+          ...sx,
+          padding: { xs: `${py ? py : '12px'} ${px ? px : '18px'}`, sm: `${py ? py : '19.5px'} ${px ? px : '20px'}` },
+        }}
+      >
+        {children}
+      </Box>
+    )
+  }
```

Comment:
I don't see any difference between the large variation and custom variation. Is it needed? Can we just tweek the existing variation because app margin shouldn't be custom but pre defined in a overall app context.